app/models.py

```diff
@@ -1,23 +1,26 @@
 """Demonstrate all models."""
 
 from werkzeug.security import generate_password_hash, check_password_hash
-from . import db
+from flask_sqlalchemy import SQLAlchemy
+
+
+db = SQLAlchemy()
 
 
 class User(db.Model):
     """Create users table."""
 
     __tablename__ = 'users'
-    uid = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.Integer, primary_key=True)
     email = db.Column(db.String(60))
     username = db.Column(db.String(60))
     first_name = db.Column(db.String(60))
     last_name = db.Column(db.String(60))
```

Comment:
Having both `username`, `first_name` and `last_name` makes using the app during sign up tedious. Having `username` and `email` should be sufficient.

Reply:
Thank you for the suggestion,
What do you think about removing `first_name` and `last_name` from sign up fields and make them optional once the user has signed up?

Reply:
That would be better.